test/getGrist.ts

```diff
@@ -183,6 +192,28 @@ export class GristUtils extends GristWebDriverUtils {
     await this.waitForServer();
   }
 
+  public async sendActions(actions: UserAction[]) {
+    const result = await driver.executeAsyncScript(`
+          const done = arguments[arguments.length - 1];
+          const prom = gristDocPageModel.gristDoc.get().docModel.docData.sendActions(${JSON.stringify(actions)});
+          prom.then(() => done(null));
+          prom.catch((err) => done(String(err?.message || err)));
+        `);
+    if (result) {
+      throw new Error(result as string);
+    }
+    await this.waitForServer();
+  }
+
+  public async waitForServer(optTimeout: number = 2000) {
```

Comment:
Is there something wrong with the existing `waitForServer`?

Reply:
Still interested in an answer to this question, what is wrong with `GristWebDriverUtils.waitForServer`?